src/Uno.UWP/System/Power/PowerManager.cs

```diff
@@ -0,0 +1,178 @@
+﻿#if __MOBILE__
```

Comment:
Let's use `__ANDROID__` and `__IOS__` explicitly, as we don't know how marzipan is going to be handled.

Reply:
Done :-)